common4j/src/main/com/microsoft/identity/common/java/opentelemetry/AttributeName.java

```diff
@@ -432,6 +447,15 @@ public enum AttributeName {
     /**
      * Records the time (in milliseconds) spent on flight check for webcp.
      */
-    web_cp_flight_get_time
+    web_cp_flight_get_time,
 
+    /**
+     * Records the Cipher transformation used to encrypt/decrypt the keypair.
```

Comment:
is it for encrypt/decrypt or wrap/unwrap?

Reply:
encrypt/decrypt https://developer.android.com/reference/javax/crypto/Cipher